### PR TITLE
Switch to the @azure/rest-api-specs-scripts@0.2.13 and convert JS to TypeScript.

### PR DESCRIPTION
**Pull request by sergey-shandar**
This PR should be merged after preproduction CI is validated in a private repo.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ script:
     if [[ $MODE == 'BreakingChange' ]]; then
       scripts/install-dotnet.sh
       npm install
+      npm install -D Azure/rest-api-specs-scripts#sergey/diff
```
Comment (nschonni) [inline .travis.yml]: Shouldn't need this every time, since it's already in package.json

Comment (sergey-shandar) [reply .travis.yml]: Don't worry about that, it's an experiment. Travis-CI didn't want to install the package properly.

**Comment by sergey-shandar** [comment]
Please, review this PR https://github.com/Azure/azure-rest-api-specs/pull/5742

**Comment by JackTn** [comment]
Hi, @sergey-shandar. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @sergey-shandar. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @sergey-shandar. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Comment by JackTn** [comment]
Hi, @sergey-shandar. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**